src/overview.md

```diff
@@ -19,16 +19,16 @@ we'll talk about that later.
 
 **TODO: someone else should confirm this vvv**
 
-- User writes a program and invokes `rustc` on it (possibly through `cargo`).
-- First, we parse command line flags, etc. This is done in [`librustc_driver`].
-  We now know what the exact work is we need to do (e.g. which nightly features
-  are enabled, whether we are doing a `check`-only build or emiting LLVM-IR or
-  a full compilation).
-- Then, we start to do compilation...
-- We first [_lex_ the user program][lex]. This turns the program into a stream
-  of _tokens_ (yes, the same sort of tokens as `proc_macros` (sort of)).
-  [`StringReader`] from [`librustc_parse`] integrates [`librustc_lexer`] with
-  `rustc` data structures.
+- The compile process begins when a user writes a Rust source program in text and invokes the `rustc` compiler on it. The work that the compiler needs to perform is defined with command line options. For example, it is possible to optionally enable nightly features, perform `check`-only builds, or emit LLVM-IR rather than complete the entire compile process defined here. The `rustc` executable call may be indirect through the use of `cargo`.
+- Command line argument parsing occurs in the [`librustc_driver`]. This crate defines the compile configuration that is requested by the user.
```

Comment:
I'm tempted to add what the configuration is turned into:
```suggestion
- Command line argument parsing occurs in the [`librustc_driver`]. This crate defines the compile configuration that is requested by the user and passes it to the rest of the compilation process as a [`rustc_interface::Config`].
```
(`rustc_interface::Config` is https://doc.rust-lang.org/nightly/nightly-rustc/rustc_interface/interface/struct.Config.html)

But I don't know if this too much of a detail to be included in the Overview...

Reply:
am happy with the change... it's not too much

Reply:
Agree but don't know this area of the code.  Can you confirm that this is the correct reference?  If not, I'll take a look this weekend and commit your suggestion here.